src/test/descriptor_tests.cpp

```diff
@@ -40,32 +43,46 @@ bool EqualDescriptor(std::string a, std::string b)
     return a == b;
 }
 
-std::string MaybeUseHInsteadOfApostrophy(std::string ret)
+std::string UseHInsteadOfApostrophe(std::string ret)
 {
-    if (InsecureRandBool()) {
-        while (true) {
-            auto it = ret.find("'");
-            if (it != std::string::npos) {
-                ret[it] = 'h';
-                if (ret.size() > 9 && ret[ret.size() - 9] == '#') ret = ret.substr(0, ret.size() - 9); // Changing apostrophe to h breaks the checksum
-            } else {
-                break;
+    while (true) {
+        auto it = ret.find("'");
+        if (it != std::string::npos) {
+            ret[it] = 'h';
+            if (ret.size() > 9 && ret[ret.size() - 9] == '#') {
```

Comment:
This will recalculate the checksum after every replacement, it would be better to just recalculate it at the end

Reply:
Thanks! Fixed in 9f9afcf1355e48d2b39ad9119415d658f5770ec5.